### ISSUE TITLE
Issues with wsdd using port causing issues and --source-port not working.

### ISSUE THREAD

**Issue by MrT-tak** [christgau/wsdd#225]
Hello!
I'm new to Linux. Apologies. Please tell me so I can learn.
It seems that WSDD is sending some kind of packet using 
a different port than 3702, which I heard is for discovery.

My computer (Win. 11) could access using the hostname, 
but it seems it broke after I accidentally broke one of the network configs.
I don't know if it is related to the known issue of the docker
interface. 

To try to fix it, I used --source-port as written in README.md (not in man page)
but it seems WSDD don't know that argument.

Firewall is UFW.

Could someone help me? Thanks!
MrT

TL;DR I'm having issues with wsdd using a different port.

**Comment by christgau** [comment]
>  I used --source-port as written in README.md (not in man page)
>  but it seems WSDD don't know that argument.

The option is [documented in the man page](https://github.com/christgau/wsdd/blob/a065cf644a361158206ffac61c2bdd16d093dc3f/man/wsdd.8#L57-L62) since a065cf644a361158206ffac61c2bdd16d093dc3f. 

Are you sure, you a are using the most recent git commit and not the version that may come with your Linux installation?

**Comment by MrT-tak** [comment]
> > I used --source-port as written in README.md (not in man page)
> > but it seems WSDD don't know that argument.
> 
> The option is [documented in the man page](https://github.com/christgau/wsdd/blob/a065cf644a361158206ffac61c2bdd16d093dc3f/man/wsdd.8#L57-L62) since [a065cf6](https://github.com/christgau/wsdd/commit/a065cf644a361158206ffac61c2bdd16d093dc3f).
> 
> Are you sure, you a are using the most recent git commit and not the version that may come with your Linux installation?

Hi @christgau,
Thanks for replying, I will look into it.
I didn't know about it, I installed WSDD via apt get so that's probably why.

I will close ASA I find out. 
Thank  you very much for the fast response!

**Comment by MrT-tak** [comment]
p.s. I later found out that that flag was on the man page of the Canonical's page. Apologies.

**Comment by MrT-tak** [comment]
@christgau, sorry, complete newbie question here.
How do I install the latest wsdd?
I installed wsdd manually initially and recently I reinstalled it via apt.
Is that not good? How should I get one from the repo?

**Comment by christgau** [comment]
>  How do I install the latest wsdd?

Sorry, but that's not the right place to ask this question. Please check your distro documentation, tutorials, forums etc. first, before directly asking the developer of certain software.

PS: `apt install wsdd` (but the version from your OS might be out of date w.r.t. to your initial problem)

**Comment by MrT-tak** [comment]
Stale. Closing.
I will request reopening when I can work on this issue.
